src/base/queue_aggregator.h

```diff
@@ -34,6 +34,7 @@ class QueueAggregator {
     for (auto& thread : threads_) {
       thread.join();
     }
+    threads_.clear();
```

Comment:
To be able to call it twice for `QueueAggregator` - used in tests. Otherwise it would try
to join already joined threads.